repository/n.json

```diff
@@ -750,6 +750,17 @@
 				}
 			]
 		},
+		{
+			"name": "NinetyFive",
+			"details": "https://github.com/ninetyfive-gg/sublime-plugin",
+			"labels": ["completions"],
```

Comment:
["ai"](https://packagecontrol.io/browse/labels/ai) is becoming popular as a label for packages like this.